buildspec.yml

```diff
@@ -7,8 +7,8 @@ phases:
     commands:
       - echo Logging in to Amazon ECR...
       - aws --version
-      - $(aws ecr get-login --region $AWS_DEFAULT_REGION --no-include-email)
       - REPOSITORY_URI=238241637211.dkr.ecr.us-west-2.amazonaws.com/speedupamerica
+      - $(aws ecr get-login-password | docker login --username AWS --password-stdin $REPOSITORY_URI)
```

Comment:
Would you remove the surrounding `$()` from this line?

Command substitution (the `$()` notation) runs its contained command in a subshell, then uses whatever that inner command prints to stdout to replace the whole command substitution string in the outer command line. For example:

```bash
$ echo foo $(echo bar) baz
foo bar baz
```

The previous version of this file used command substitution because `aws ecr get-login` prints a `docker login` command line to stdout and we needed to *run* that printed command. In your new version, though, you're running `docker login` yourself and we don't want to run its output as a command (`Login Succeeded` isn't a valid command).